Sources/WordPressKit/Models/Stats/Emails/StatsEmailOpensData.swift

```
@@ -0,0 +1,42 @@
+import Foundation
+
+public struct StatsEmailOpensData: Decodable, Equatable {
+    public let totalSends: Int?
+    public let uniqueOpens: Int?
+    public let totalOpens: Int?
+    public let opensRate: Double?
+
+    public init(totalSends: Int?, uniqueOpens: Int?, totalOpens: Int?, opensRate: Double?) {
+        self.totalSends = totalSends
+        self.uniqueOpens = uniqueOpens
+        self.totalOpens = totalOpens
+        self.opensRate = opensRate
+    }
+
+    private enum CodingKeys: String, CodingKey {
+        case totalSends = "total_sends"
+        case uniqueOpens = "unique_opens"
+        case totalOpens = "total_opens"
+        case opensRate = "opens_rate"
+    }
+
+    public init(from decoder: any Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        totalSends = try container.decodeIfPresent(Int.self, forKey: .totalSends)
+        uniqueOpens = try container.decodeIfPresent(Int.self, forKey: .uniqueOpens)
+        totalOpens = try container.decodeIfPresent(Int.self, forKey: .totalOpens)
+        opensRate = try container.decodeIfPresent(Double.self, forKey: .opensRate)
+    }
+}
+
+extension StatsEmailOpensData {
+    public init?(jsonDictionary: [String: AnyObject]) {
```

`throw` instead of silencing the error?

---

I auto-generated this code with the rest of the file as a reference. It is consistent with the rest of `StatsServiceRemoteV2` and how it's used. I'd suggest keeping it as it's throw-away code anyway.